Lib/fontTools/varLib/instancer/__init__.py

```diff
@@ -1443,7 +1448,7 @@ def main(args=None):
             axisLimits = instance.coordinates
             log.info("Restricting axes: %s (%s)", axisLimits, name)
             name = name.replace(" ", "")
-            outfile = os.path.splitext(infile)[0] + "-" + name + ".ttf"
+            outfile = outprefix + "-" + name + ".ttf"
```

Comment:
this is not exactly what Khaled and I had in mind though, namely

> If --output is used this code should error, or, if it wants to be smart, use it for the first matching instance and error for the rest, so one can use it to output a single named instance

Reply:
if you want to control where multiple outputs go, i think it'd be better to add a new explicit --output-dir option